databases/week2/part3.sql

```diff
@@ -0,0 +1,15 @@
+USE hyf_class2023;
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT task.id, task.title, user.email FROM task INNER JOIN user ON task.user_id = user.id WHERE user.email LIKE '%@spotify.com';
+-- Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT task.id, task.title, task.created, user.name FROM task
+INNER JOIN status ON task.status_id = status.id
+INNER JOIN user ON task.user_id = user.id
+WHERE status.name='Not started' AND user.name='Donald Duck';
+-- Get all the tasks for 'Maryrose Meadows' that were created in september (hint: month(created)=month_number)
+SELECT task.id, task.title, task.created, user.name FROM task
+INNER JOIN user ON task.user_id = user.id
+WHERE MONTH(task.created)=9 AND user.name='Maryrose Meadows';
```

Comment:
Pls, be aware you are not using table `user_task`  where necessary & your query doesn't make any result...
```
SELECT task.id,user.name, task.title, task.created
FROM task
JOIN user_task ON task.id = user_task.task_id
JOIN user  ON user.id = user_task.user_id
WHERE user.name LIKE 'Maryrose Meadows' AND MONTH(created)=09;
```